pom.xml

```diff
@@ -72,10 +93,10 @@
             <version>${feign.extension.version}</version>
          </dependency>
          <dependency>
-         <groupId>com.vaadin</groupId>
-         <artifactId>vaadin-spring-boot-starter</artifactId>
-         <version>${vaadin.spring.version}</version>
-      </dependency>
+            <groupId>com.vaadin</groupId>
```

Comment:
Are we still using this dependency in one of our examples? Since even `hawkbit-custom-theme-example` is not using it we should remove it.